src/components/components/event/event.js

```diff
@@ -21,6 +21,33 @@ const Event = props => {
   const endDateElement = endDate && endDate !== startDate && (
     <span className="event__date-end">{shortDate(endDate)}</span>
   );
+
+  const location =
```

Comment:
could you split that to another component? I looks like it would simplify the event component a lot. Also we have a similar functionality inside the club component, maybe this one and that one can be turned into a single component